python/vyos/utils/system.py

```diff
@@ -139,3 +139,11 @@ def get_load_averages():
     res[15] = float(matches["fifteen"]) / core_count
 
     return res
+
+def get_secure_boot_state() -> bool:
+    from vyos.utils.process import cmd
+    from vyos.utils.boot import is_uefi_system
+    if not is_uefi_system():
+        return False
+    tmp = cmd('mokutil --sb-state')
```

Comment:
This line throws an exception for devices that do not support secure-boot.  should this be wrapped in a try?

```
# show ver
Traceback (most recent call last):
  File "/usr/libexec/vyos/op_mode/version.py", line 91, in <module>
    res = vyos.opmode.run(sys.modules[__name__])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/vyos/opmode.py", line 271, in run
    res = func(**args)
          ^^^^^^^^^^^^
  File "/usr/libexec/vyos/op_mode/version.py", line 81, in show
    version_data = _get_raw_data(funny=funny)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/libexec/vyos/op_mode/version.py", line 67, in _get_raw_data
    if get_secure_boot_state():
       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/vyos/utils/system.py", line 148, in get_secure_boot_state
    tmp = cmd('mokutil --sb-state')
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/vyos/utils/process.py", line 155, in cmd
    raise OSError(code, feedback)
OSError: [Errno 255] failed to run command: mokutil --sb-state
returned:
exit code: 255

# mokutil --sb-state
This system doesn't support Secure Boot
```